src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.tsx

```diff
@@ -35,11 +35,11 @@ export interface IVoteDataListItemStructureProps extends IDataListItemProps {
     tokenSymbol?: string;
 }
 
-const voteIndicatorToTagVariant: Record<IVoteDataListItemStructureProps['voteIndicator'], TagVariant> = {
+export const voteIndicatorToTagVariant: Record<IVoteDataListItemStructureProps['voteIndicator'], TagVariant> = {
     yes: 'success',
     no: 'critical',
     abstain: 'neutral',
-    approved: 'primary',
+    approve: 'primary',
 };
```

Comment:
I would move this constant and a VoteIndicator type to a new file under the `/vote` folder, like:
```
[/components/vote/voteUtils.ts]

export type VoteIndicator = 'yes' | 'no' | 'abstain' | 'approve';

export const voteIndicatorToTagVariant: [...]
```
And use it on both the vote components